dev/README_RELEASE_AIRFLOW.md

```diff
@@ -352,7 +352,7 @@ is not supposed to be used by and advertised to the end-users who do not read th
     (both airflow and latest provider packages).
 
     ```shell script
-    git push origin ${VERSION}
+    git push origin tag ${VERSION}
```

Comment:
The original should work too

Reply:
Indeed, but this is safer.

Reply:
TIL this is a thing

Reply:
Git never ceases to amaze me. Every time I get to do something really complex in git, it really is just a matter of looking hard enough. There has ALWAYS been a one liner (even if a line is long) that does precisely what I wanted to do - no matter how complex the thing was.